jupyter_scheduler/scheduler.py

```diff
@@ -608,6 +613,17 @@ def list_job_definitions(self, query: ListJobDefinitionsQuery) -> ListJobDefinit
 
         return list_response
 
+    def create_job_from_definition(self, job_definition_id: str, model: CreateJobFromDefinition):
+        job_id = None
+        definition = self.get_job_definition(job_definition_id)
+        if definition and definition.active:
```

Comment:
My thought is that the run now from a job def should work even if the job def is paused. Thoughts?

Reply:
I think run now should work even if the job def is paused as I can see scenarios where users would want to run a job off paused definition without restarting it (e.g. if it's a resource/time-heavy job with a high-frequency schedule)

Reply:
Agree, we are reusing this from task runner as well, so this was a copy. Will correct this.

Reply:
Updated.